include/relic_multi.h

```diff
@@ -56,6 +56,8 @@
  */
 #if MULTI == PTHREAD
 #define rlc_thread 	__thread
+#elif MULTI == MSVCTLS
+#define rlc_thread 	__declspec( thread )
```

Comment:
this was standardized in C11 as `thread_local`. https://en.cppreference.com/w/c/thread/thread_local